tests/_data/simple_multipart.txt

```diff
@@ -1,30 +1,30 @@
-User-Agent: curl/7.21.2 (x86_64-apple-darwin)
-Host: localhost:8080
-Accept: */*
-Content-Length: 1143
-Expect: 100-continue
-X-Multi-Line: line one
-    line two with space
-	line three with tab
-Content-Type: multipart/form-data; boundary=----------------------------83ff53821b7c
-
-------------------------------83ff53821b7c
-Content-Disposition: form-data; name="img"; filename="a.png"
-Content-Type: image/png
-
-?PNG
-
-IHD?wS??iCCPICC Profilex?T?kA?6n??Zk?x?"IY?hE?6?bk
-Y?<ߡ)??????9Nyx?+=?Y"|@5-?M?S?%?@?H8??qR>?׋??inf???O?????b??N?????~N??>?!?
-??V?J?p?8?da?sZHO?Ln?}&???wVQ?y?g????E??0
- ??
-   IDAc????????-IEND?B`?
-------------------------------83ff53821b7c
-Content-Disposition: form-data; name="foo"
-
-bar
-------------------------------83ff53821b7c
-Content-Disposition: form-data; name="rfc5987"; text1*=iso-8859-1'en'%A3%20rates; text2*=UTF-8''%c2%a3%20and%20%e2%82%ac%20rates
-
-rfc
-------------------------------83ff53821b7c--
+User-Agent: curl/7.21.2 (x86_64-apple-darwin)
```

Comment:
this files seems changed. The encoding here was important.

Reply:
probably thats why is failing

Reply:
Fixed. Previously I renamed directory  by ide, now renamed by `mv` terminal command.
It should be ok now.